CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [4.0.0-alpha.1](https://github.com/dtassone/material-ui-x/compare/v0.1.67...v4.0.0-alpha.1) (2020-09-17)
+
+**Note:** Version bump only for package material-ui-x
```

Comment:
So far, I have used this script to generate the changelog

https://trello.com/c/uspnIWkh/1566-release-note

Running on this page: https://github.com/mui-org/material-ui-x/compare/v0.1.67...master

Reply:
What does this script do? How do you run it?

Reply:
It's one way to convert the git history into a changelog:

<img width="1122" alt="Capture d’écran 2020-09-17 à 15 01 54" src="https://user-images.githubusercontent.com/3165635/93473803-c04aeb00-f8f6-11ea-877b-71796193c9c3.png">

Reply:
![image](https://user-images.githubusercontent.com/936978/93474200-541cb700-f8f7-11ea-93b1-a6f568530d60.png)

Reply:
Wrong page:

<img width="290" alt="Capture d’écran 2020-09-17 à 15 21 02" src="https://user-images.githubusercontent.com/3165635/93475865-6a2b7700-f8f9-11ea-8ea1-e77f1a542589.png">